src/config/sitemap.js

```diff
@@ -65,7 +66,10 @@ async function getSitemapRoutes() {
   const portfolioPageRoutes = [...new Set(creator.concat(collectors))].map(
     id => `/${id}`
   );
-  return nftDetailsPageRoutes.concat(portfolioPageRoutes);
+  return nftDetailsPageRoutes.concat(portfolioPageRoutes).map(url => ({
+    url: `/${locales[0]}${url}`,
```

Comment:
???
```suggestion
    url: `/${locales[0]}/${url}`,
```

Reply:
Our `url` starts with / as shown in e.g. line 66